content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -150,8 +150,8 @@ the possible following fields:
   (the file's location).
 - `deps`: List of <abbr>dependency</abbr> file or directory paths of this stage
   (relative to `wdir` which defaults to the file's location)
-- `params`: List of [parameter dependencies](/doc/command-reference/params).
-  These are key paths referring to a YAML, JSON or TOML file (`params.yaml` by
+- `params`: List of <abbr>parameter<abbr> dependencies. They are key/value pairs
```

Comment:
they are not values though, just keys, right?

Reply:
Ah yes I was a bit confused with dvc.loc. BTW I just updated the dvc.lock part as well accordingly. See d5e7a468.